invoice/src/main/resources/org/killbill/billing/invoice/migration/V20190121141325__tracking_ids_is_active.sql

```diff
@@ -0,0 +1,4 @@
+alter table invoice_tracking_ids add column is_active boolean default true after record_date;
+alter table invoice_tracking_ids add column updated_by varchar(50) NOT NULL after created_date;
+alter table invoice_tracking_ids add column updated_date datetime NOT NULL DEFAULT '1970-01-01 00:00:00' after updated_by;
+create index invoice_tracking_invoice_id_idx on invoice_tracking_ids(tenant_record_id, invoice_id);
```

Comment:
Shouldn't `invoice_id` come first (bigger cardinality)?

Reply:
This is not really clear. From this [post](http://colleenmorrow.com/2011/04/07/composite-indexes-does-column-order-matter/):
```
I agree that on composite indexes, the choice of ordering the most selective first doesn’t normally impact performance. This is contrary to what is often given as advice....
```

Besides perf, the other aspect in the index ordering is whether we want to use this index for queries using only one column, in which case, this column needs to be first in the index.

Stepping back, since our `invoiceId` are unique across tenants, i could simplify and only have an index on the `invoice_id`  -- we already have a composite index `tenant_record_id, account_record_id` allowing to search per tenant if required. Thoughts?